counterparty-core/counterpartycore/lib/telemetry/util.py

```diff
@@ -0,0 +1,43 @@
+import os
```

Comment:
What ReST endpoint should we expose this at? (@ouziel-slama)

Reply:
hum.. for now I put the equivalent of `get_running_info` at the root path '/'. We could have `/server` resource also.

Reply:
I think the root path might be sufficient! Unless there's some standard I don't know about...

Reply:
yep, that sounds good!

Reply:
ah great! I didn't see the server name